source/release-notes/1.8.txt

```diff
@@ -21,7 +21,18 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
```

Comment:
Should we remove Upcoming in this PR and instead put the release date (today)? Or do we do this in a follow up?

Reply:
I'll tie that in with this PR. Updated!